uber/sep_commands.py

```diff
@@ -12,6 +12,17 @@ def print_config():
 
 
 @entry_point
+def resave_all_attendees_and_groups():
+    Session.initialize_db(modify_tables=True)
+    with Session() as session:
+        print("re-saving all attendees....")
+        [a.presave_adjustments() for a in session.query(Attendee).all()]
+        print("re-saving all groups....")
+        [g.presave_adjustments() for g in session.query(Group).all()]
+        print("Done!")
```

Comment:
yea I noticed that too, it's also not til we exit the with block that the commit takes effect, so it prints "done" and then takes 20 seconds to actually do the commit. will fix